src/ui/mainwindow.cpp

```diff
@@ -455,6 +471,36 @@ void MainWindow::createTab()
     m_tabBar->setCurrentIndex(index);
 }
 
+void MainWindow::duplicateTab()
```

Comment:
A lot of code is taken from `MainWindow::createTab()`. It would be nice to avoid this code duplication.